stellargraph/core/graph.py

```diff
@@ -157,6 +157,15 @@ class StellarGraph:
         0 of type ``a``, and a node 0 of type ``b``. IDs of edges must also be unique across all
         types.
 
+    .. _iloc-explanation:
+
+        This type stores the external IDs for nodes and edges as *ilocs* (internal or integer locations,
```

Comment:
Is this indentation on purpose? I think it means the paragraph looks a bit peculiar because it's not aligned? https://stellargraph--1220.org.readthedocs.build/en/1220/api.html#iloc-explanation

![image](https://user-images.githubusercontent.com/1203825/79393788-647fdd00-7fb9-11ea-9dfb-f5c68236532b.png)

Reply:
i was just copynig the identatino of "Note" above but it looks super weird. I've removed the indent